### DISCUSSION TITLE
How can I render as raw text for not allowed elements in react markdown?

### DISCUSSION THREAD

**Discussion by sungwoo91** [remarkjs/remark#1235] [Q&A] [answered]
Hi, 

I'm using react-markdown and retype-raw.

I have a question, I think the allowed elements option in react markdown with rehype-raw does not render anything other than allowed elements. however, I would like to show this part as a plain text without rendering it in the html engine.

Is there a way to show elements in raw text for not allowed elements?

**Comment by ChristianMurphy** [comment, marked as answer]
Cross referencing https://github.com/remarkjs/react-markdown/issues/408 which has some previous discussion.
To recap:
1. If you want to make the rendered content look more like the source markdown, consider using `components` https://github.com/remarkjs/react-markdown#appendix-b-components instead of `allowedElements` to custom render the output.
2. If you want to skip parsing some parts of markdown
   * It is generally **_not advisable to do this_**. This essentially creates a new Markdown flavor, it can and likely will confuse content authors, for more info on the trade offs and alternatives see: https://github.com/micromark/micromark#extending-markdown
   * If you do choose to make you own Markdown flavor anyway, there is documentation on how to disable constructs (syntax features) here: https://github.com/micromark/micromark#case-turn-off-constructs
